glean-core/src/metrics/labeled.rs

```diff
@@ -2,15 +2,14 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-use std::collections::HashSet;
-
+//! Module documentation
```

Comment:
nit: is this a placeholder?

Reply:
Yes -- in an earlier version I made this whole module public which then required a docstring.  I rolled that back so this should just be removed.